### PR TITLE
CC-551 card_declined failure reason

### PR DESCRIPTION
**Pull request by kgeorgiev**

### CODE REVIEW

File: source/reference/v2/payments-api/get-payment.rst [hunk 1]
```diff
@@ -788,7 +788,7 @@ Credit card
 
               Possible values: ``authentication_failed``  ``card_expired`` ``inactive_card`` ``insufficient_funds``
               ``invalid_card_holder_name`` ``invalid_card_number`` ``invalid_card_type`` ``invalid_cvv``
-              ``possible_fraud`` ``refused_by_issuer`` ``unknown_reason``
+              ``possible_fraud`` ``refused_by_issuer`` ``card_declined`` ``unknown_reason``
```
Comment (mvdpanne) [inline source/reference/v2/payments-api/get-payment.rst:791]: I think the values should be alphabetically sorted here.

**Comment by willemstuursma** [comment]
What's the difference between `card_declined` and `refused_by_issuer`?

**Comment by mvdpanne** [comment]
@willemstuursma Bart wanted another generic reason that does not mention the issuer, as not all cases relate to the issuer.

He preferred this over using the `unknown_reason`, as the message that is displayed for that failure reason indicates that we don't know why the card was declined and that the shopper should contact their bank for more information. That's not what Bart wanted us to convey.

File: source/reference/v2/payments-api/get-payment.rst (hunk 1 above)
Comment (mollierobbert) [reply source/reference/v2/payments-api/get-payment.rst]: Done.

**Pull request merged**